packages/massa-proto-cli/README.md

```diff
@@ -0,0 +1,95 @@
+# Massa-Proto-CLI
+
+Massa-Proto-CLI is a tool designed to generate callable functions for smart contract methods in TypeScript and AssemblyScript.
+These generated functions can be seamlessly integrated within your smart contracts from TypeScript or AssemblyScript projects.
+
+## Getting Started
+
+### Prerequisites
+Ensure you have npm installed on your machine to run the necessary commands.
+
+For AssemblyScript (AS) smart contract caller generation, run:
+
+```bash
+npm i as-proto as-proto-gen @massalabs/massa-as-sdk @massalabs/as-types @massalabs/as-transformer
+```
+
+For TypeScript (TS) web3 caller generation, run:
+
+```bash
+npm i @protobuf-ts/plugin
+```
+
+
+### Installation
+TODO
```

Comment:
You need to clarify how to install. The concept of nightly is not used for the other project.